test/javascripts/acceptance/core-sidebar-test.js

```diff
@@ -14,11 +14,12 @@ import { emojiUnescape } from "discourse/lib/text";
 import User from "discourse/models/user";
 
 acceptance("Discourse Chat - Core Sidebar", function (needs) {
-  needs.user({ experimental_sidebar_enabled: true, has_chat_enabled: true });
```

Comment:
Hmm is there a reason why the test didn't fail when the setting was no longer valid?

Reply:
It was failing, I just fixed it quickly to have green CI. I have to add test that old sidebar is not visible (that will be next PR)

Reply:
ahh ok :+1: